provider/keycloak_openid_client_test.go

```diff
@@ -129,6 +129,10 @@ func TestAccKeycloakOpenidClient_updateInPlace(t *testing.T) {
 	directAccessGrantsEnabled := randomBool()
 	serviceAccountsEnabled := randomBool()
 
+	if !standardFlowEnabled {
+		implicitFlowEnabled = !standardFlowEnabled
+	}
```

Comment:
What's the reason for this change?

Reply:
At least standard or implicit flow should be enabled to set valid redirect URI

Reply:
Is that new behavior with version 5.0.0 or 6.0.0 of Keycloak.  I realize that there is no point in setting valid redirect uris without either of these two flows enabled, but I don't believe Keycloak 4.8.3 requires it.

I'm fine with keeping this change, I was just curious what the context was.